bin/ch-checkns.c

```diff
@@ -52,6 +52,7 @@
 
 
 const char usage[] = "\
+\n\
```

Comment:
Why a newline added here?

Reply:
The newline was intended to make the message similar to the other which have a newline there as well. The message is quite large, and I think it improves readability.